<ui version="4.0" >
 <class>control</class>
 <widget class="QWidget" name="control" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ToolBox</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>10</y>
     <width>161</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title" >
    <string>Objects</string>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string>Click on a object to select it
Current Poly:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>85</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>N:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Kd (Also color):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>140</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ka</string>
    </property>
   </widget>
   <widget class="QLabel" name="asdf" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>180</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Kr</string>
    </property>
   </widget>
   <widget class="QSlider" name="kr" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>200</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sliderPosition" >
     <number>50</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>220</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ks</string>
    </property>
   </widget>
   <widget class="QSlider" name="ks" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>240</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sliderPosition" >
     <number>50</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="asdf_2" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>260</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Kt</string>
    </property>
   </widget>
   <widget class="QSlider" name="kt" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>280</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sliderPosition" >
     <number>50</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="nEdit" >
    <property name="geometry" >
     <rect>
      <x>105</x>
      <y>50</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selected" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>50</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="ka" >
    <property name="geometry" >
     <rect>
      <x>15</x>
      <y>160</y>
      <width>129</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sliderPosition" >
     <number>25</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_6" >
    <property name="geometry" >
     <rect>
      <x>16</x>
      <y>110</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="kdR" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kdG" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kdB" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Global</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Const C:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ambient Intensity (Color):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iaG" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iaB" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="c" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iaR" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>Lights</string>
   </property>
   <widget class="QLabel" name="numLights" >
    <property name="geometry" >
     <rect>
      <x>61</x>
      <y>20</y>
      <width>13</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Direction Vector:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>147</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lightR" >
       <property name="text" >
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightG" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightB" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Light Intensity (Color):</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lightx" >
       <property name="text" >
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lighty" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lightz" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>150</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string>There are     lights in this scene.
Choose one to edit.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lightEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>320</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string>Camera</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="cameradx" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="camerady" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cameradz" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="upx" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="upy" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="upz" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_12" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Direction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Up:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Origin:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="camerax" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cameray" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cameraz" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>520</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>350</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Rendering:</string>
   </property>
   <widget class="QLabel" name="label_14" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resolution" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Max Depth:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxDepth" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>40</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="traceButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>520</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ray Trace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
